.github/workflows/release.yml

```
@@ -0,0 +1,97 @@
+name: Release
+on:
+  push:
+    tags:
+      - '[2-9]+.[0-9]+.[0-9]+'
+      - '[2-9]+.[0-9]+.[0-9]+-*'
```

```suggestion
      - '[2-9].[0-9]+.[0-9]+'
      - '[2-9].[0-9]+.[0-9]+-*'
```

I think we need to remove the `+` quantifier as this shouldn't match on multi-digit numbers (e.g. that would match on 24.0 but not on 10.0). From what I saw, other release definitions also don't use the +, e.g.
* https://github.com/Particular/NServiceBus/blob/master/.github/workflows/release.yml
* https://github.com/Particular/NServiceBus.AmazonSQS/blob/master/.github/workflows/release.yml
* https://github.com/Particular/NServiceBus.RavenDB/blob/master/.github/workflows/release.yml